package.json

```diff
@@ -165,6 +173,16 @@
           "command": "code-annotation.editNote",
           "when": "view == codeAnnotationView && viewItem == $CompleteNote",
           "group": "3_open@2"
+        },
+        {
+          "command": "code-annotation.copyNote",
+          "when": "view == codeAnnotationView && viewItem == $PendingNote",
```

Comment:
If you do just `"when": "view == codeAnnotationView"` you won't need both configurations.